extensions/word-count/src/count.tsx

```diff
@@ -9,15 +10,19 @@ export default function Command() {
 
   useEffect(() => {
     (async () => {
-      const clipboard = await Clipboard.readText();
+      try {
+        const selectedText = await getSelectedText();
 
-      if (clipboard) {
-        setText(clipboard.trim());
-        showToast({
-          style: Toast.Style.Success,
-          title: `Text loaded from clipboard`,
-          message: `[⌘ + E] to reset`,
-        });
+        if (selectedText) {
+          setText(selectedText.trim());
+          return;
+        }
+
+        const clipboard = await readFromClipboard();
+        setText(clipboard);
+      } catch {
+        const clipboard = await readFromClipboard();
+        setText(clipboard);
```

Comment:
Not a huge fan of blending the two steps into the same try-catch block. Would something like this make the steps easier to comprehend? 

```ts
      let content = "";

      try {
        const selectedText = await getSelectedText();
        if (selectedText) {
          content = selectedText.trim();
        }
      } catch {
        // ignore error, fallback to clipboard
      }

      if (!content) {
        content = await readFromClipboard();
      }

      setText(content);
```